prediction_market_agent/agents/microchain_agent/deploy.py

```diff
@@ -0,0 +1,22 @@
+from microchain import Agent
+from prediction_market_agent_tooling.deploy.agent import DeployableAgent
+from prediction_market_agent_tooling.markets.markets import MarketType
+
+from prediction_market_agent.agents.microchain_agent.microchain_agent import get_agent
+
+
+class DeployableMicrochainAgent(DeployableAgent):
+    model = "gpt-4-1106-preview"
+    n_iterations = 50
+
+    def run(self, market_type: MarketType, _place_bet: bool = True) -> None:
+        """
+        Override main 'run' method, as the all logic from the helper methods
+        is handed over to the agent.
+        """
+        agent: Agent = get_agent(
+            market_type=market_type,
+            model=self.model,
+        )
+        agent.bootstrap = [f'Reasoning("I need to reason step by step")']
```

Comment:
Why is the `.bootstrap = ` always separate from `get_agent` or from the agent's initialisation? Feels like it could be just one step. (but even if yes, no need to edit in this PR)

Reply:
Good point. Have moved `.bootstrap =` inside `get_agent`